README.md

```diff
@@ -1,5 +1,7 @@
 # [dynamiclistener](https://github.com/rancher/dynamiclistener)
 
+DynamicListener allows you to setup a server with automatically generated (and re-generated) TLS certs with kubernetes secrets integration.
```

Comment:
Added this to fix an issue with drone which is associating this PR with the k8s-1.30 branch and running FOSSA unintentionally.